sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -133,15 +133,15 @@ public void testSelectCountStart() throws Exception
         CalciteTests.REGULAR_USER_AUTH_RESULT,
         ImmutableList.of(Druids.newTimeseriesQueryBuilder()
                                .dataSource(CalciteTests.DATASOURCE1)
-                               .intervals(QSS(Filtration.eternity()))
-                               .filters(SELECTOR("dim2", "0", null))
+                               .intervals(qss(Filtration.eternity()))
```

Comment:
Hmm, so I think I actually prefer these test shorthand methods as ALLCAPS instead of lowercase, makes it to parse for me at least. Was the intention to disallow all uppercase static methods too or just methods that started with an uppercase letter and were not fully uppercase?

If no one agrees with me feel free to disregard this review 🙃

Reply:
@clintropolis I think the intention was to restrict any method names starting with a capital letter and that covered all the static methods (with Caps) as well.

Reply:
@clintropolis in my opinion, methods names that are acronyms is a bad idea. I think better to expand them.

Reply:
No dispute here that those methods names are not great, but I guess I can see why they were done initially that way since those tests make big walls of text (not an argument to keep them).

Should we just rename them to sensible names that spell out the type they are making in this PR instead of renaming them to be lowercase and then having to rename them all again later?

Reply:
I don't have any preference here. Both works, I think. But not making an exception in the rule to enable all caps names.

Reply:
Are we going to changes these names then?

Reply:
To me, the PR is fine in its current state (but conflicts should be fixed), unabbreviating method names would be nice but not required in this PR.

Reply:
Thanks! Resolved those.

Reply:
If you don't mind, please un-abbreviate the method names in the calcite tests since you're making changes to all of these lines anyway might as well make a better change, and switch them from snake_case to camelCase where possible, e.g. `expresion_Post_Agg` (which also has a typo) to `expressionPostAgg`.

Additionally, somehow the commit history on this PR has gotten very messed up when you fixed conflicts (at least on github) and it's showing over 25k added lines, making it basically impossible to review this PR anymore. I'm not sure how it got into this state, but it might be better to cherry pick your commits from this branch into another branch and open a new PR if you can't find a way to fix it.

Also please don't mark review conversations as resolved, it makes it hard for the reviewer to follow up on things.

Reply:
@clintropolis sure. Let me fix it with a new PR. Thanks!

Reply:
@clintropolis @leventov  created a new PR https://github.com/apache/incubator-druid/pull/7118/